db/seeds.rb

```diff
@@ -0,0 +1,13 @@
+Post.create!(name:"1st blog post", title:" this is the title", content:"this is the content")
+
+Comment.create!(name:"guest comment", commField: "this is a rad post", post_id: 1)
```

Comment:
Comments don't usually exist absent of a post. How could you change your seeds file so that this comment belongs to the post that is created before it?